src/util/unitOfMass.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Represents units of mass, e.g. Kilogram, the base unit of mass.
+ * @enum
+ */
+export enum UnitOfMass {
```

Comment:
if you only want to support kg, what's the point of this?

Reply:
Yeah, you're right.. I took care of it / removed it.